src/main/scala/GameLib.scala

```diff
@@ -1,33 +1,72 @@
 package strategygames
 
 sealed abstract class GameLib {
-  //def white: Color
-  //def black: Color
   def id: Int
   def name: String
 
   override def toString = s"Lib($name)"
 }
 
 object GameLib {
+
   final case class Chess() extends GameLib {
-    //def white = Color.Chess(chess.White)
-    //def black = Color.Chess(chess.Black)
     def id = 0
     def name = "Chess"
   }
+
   final case class Draughts() extends GameLib {
-    //def white = Color.Draughts(draughts.White)
-    //def black = Color.Draughts(draughts.Black)
     def id = 1
     def name = "Draughts"
   }
 
-  def all = List(Draughts, Chess)
+  def all = List(Chess, Draughts)
 
   // TODO: I'm sure there is a better scala way of doing this
   def apply(id: Int): GameLib = id match {
-    case 0 => Chess()
-    case _ => Draughts()
+    case 1 => Draughts()
+    case _ => Chess()
+  }
+}
+
+sealed abstract class DisplayLib {
```

Comment:
Should DisplayLib be in its own file? If you are trying to distinguish the difference between this and GameLib having them in the same file maybe confusing in the future? I'm not sure of the current setup/structure yet.

Reply:
I feel like they are related enough that they can live together, and there are other instances such as `Divider.scala` which are similar to this

Reply:
They're used throughout, they could be defined in the package.scala or somewhere more general too. I like having them in the same file, but I agree with James that having them inside of a "GameLib" file is somewhat confusing.

Reply:
I'm planning to move more stuff into these files when I move stuff out of lila, so I think it should be separate from package.scala. Happy to rename the file to something but what would you then call that?